app/templates/main/record.html

```diff
@@ -108,8 +108,9 @@ <h1 class="govuk-heading-m govuk-heading-m__record-details">Record details</h1>
                 <div class="govuk-grid-column-one-third govuk-grid-column-one-third--record">
                     <div class="rights-container">
                         <h3 class="govuk-heading-m govuk-heading-m__rights-header">Rights to access</h3>
-                        <button class="govuk-button govuk-button__download--record"
-                                data-module="govuk-button">Download record</button>
+                        <a href="{{ url_for('main.download_record', record_id=record['file_id']) }}"
```

Comment:
syling needs fixing but not sure exactly whats the issue here, worth chatting with @colinbowen and @zaiziclebby

Reply:
Fixed in the latest commit